packages/content/test/attributes/EmailAddress.test.ts

```diff
@@ -0,0 +1,59 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { EMailAddress } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type EMailAddress", function () {
+    test("can create a RepositoryAttribute of Attribute Value Type EMailAddress", function () {
+        const validEMailAdress = EMailAddress.from({
+            value: "peter123@inwind.it"
+        });
+        expect(validEMailAdress.value.toString()).toBe("peter123@inwind.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type EMailAddress with the German 'ä' in its domain", function () {
+        const validEMailAdress = EMailAddress.from({
+            value: "peter123@inwänd.it"
+        });
+        expect(validEMailAdress.value.toString()).toBe("peter123@inwänd.it");
+    });
+    test("returns an error when trying to create an Attribute Value Type EMailAddress with a blank in the value for EMailAddress", function () {
+        const invalidEMailAdressCall = () => {
```

Comment:
`invalidEMailAddressCall` instead of `invalidEMailAdressCall`.

Same in other places below.